docs/tutorial/part-two/index.md

```diff
@@ -14,32 +14,32 @@ disableTableOfContents: true
 
 -->
 
-Welcome to part two of the Gatsby tutorial!
+Selamat datang di bagian dua dari tutorial Gatsby!
 
-## What's in this tutorial?
+## Apa yang ada di dalam tutorial ini?
 
-In this part, you're going to explore options for styling Gatsby websites and dive deeper into using React components for building sites.
+Pada bagian ini, Anda akan menjelajahi pilihan untuk _styling_ situs web Gatsby dan menyelam lebih dalam tentang penggunaan komponen React untuk membangun situs.
 
```

Comment:
Please use italic in english language context

Reply:
do you mean web ?

Reply:
Sorry, i mean line 23